test/runtests.jl

```diff
@@ -250,8 +250,32 @@ end
 end
 
 module ImportBase; import Base.push!; push!(::Int)=1; end
+
+module NonImportBase; Base.push!(::String)="1"; end
+
+module RandomMod; f() = nothing; end
+
+module AddMethodsAllOverThePlace
+    struct MyStruct end
+    Core.println(::MyStruct) = 1
+    Base.push!(::MyStruct) = 1
+    my(::MyStruct) = 1
+    using ..RandomMod
+    RandomMod.f(::MyStruct) = 1
+    using Test
+    Test.finish(::MyStruct) = 1
+    using StabilityCheck
+    StabilityCheck.checkModule(::MyStruct) = 1
+end
+
 @testset "Method discovery completeness   " begin
     chks = is_stable_module(ImportBase)
     @test length(chks) == 1
     @test chks[1].check == Stb(1)
+    chks = is_stable_module(NonImportBase; extra_modules=[Base])
```

Comment:
It looks like `[Base]` could be a better default than `[]`?

Reply:
I thought it's better to leave this explicit, i.e., by default you only get functions you find in `m`. But I can change it if you want - maybe `[Base, Core]` would make sense then?

Reply:
I love the description in README, thanks! Why not `Base.loaded_modules_array()` as the default? It seems "more correct" and not too expensive?..